packages/ui/src/utils/get-user-updated-properties-schema.ts

```diff
@@ -0,0 +1,41 @@
+import { isDefined } from '.';
```

Comment:
It is better to reference the file directly, rather than using the barrel file (`index.ts`)
```suggestion
import { isDefined } from './is-defined.ts';
```